source/draft_api.rst

```diff
@@ -297,3 +297,41 @@ If the object does not exist, the API will return an Object Not Found error
 
 In case of any other error, the API will return a Server Error (see error
 docs).
+
+-----------------------
+
+Authorization and Roles
+-----------------------
+
+Each timesync user can be of one of two roles: user, and admin. In addition, each user
+has a role within each project to which they belong: member, data viewer, and project
+manager. Each role is cumulative: that is, a data viewer has all of the abilities that a
```

Comment:
do we want viewers to be able to write? that seems like [Grade A Misleading](http://i.imgur.com/XHMhxcr.png), and a Very Bad Idea.